src/Microsoft.Azure.SignalR.Common/Utilities/RestClient.cs

```diff
@@ -37,47 +38,54 @@ public RestClient(IHttpClientFactory httpClientFactory, IPayloadContentBuilder c
     public Task SendAsync(
         RestApiEndpoint api,
         HttpMethod httpMethod,
-        string? methodName = null,
-        object[]? args = null,
-        Func<HttpResponseMessage, bool>? handleExpectedResponse = null,
-        CancellationToken cancellationToken = default)
-    {
-        return handleExpectedResponse == null
-            ? SendAsync(api, httpMethod, methodName, args, handleExpectedResponseAsync: null, cancellationToken)
-            : SendAsync(api, httpMethod, methodName, args, response => Task.FromResult(handleExpectedResponse(response)), cancellationToken);
-    }
+        CancellationToken cancellationToken = default) =>
```

Comment:
Why do we remove `methodName` and `args` parameter

Reply:
they are not used.